provisioning/ubuntu-provision.sh

```diff
@@ -355,6 +355,33 @@ function install_doctl(){
     "https://github.com/digitalocean/doctl/releases/download/v${DOCTL_VERSION}/doctl-${DOCTL_VERSION}-linux-${ARCHITECTURE}.tar.gz" | tar --extract --gunzip --directory="${install_dir}"/ doctl
 }
 
+## Ensure that HELM and it's plugins are installed
+function install_helm(){
+  install_dir=/usr/local/bin
+  curl --fail --silent --location --show-error \
+    "https://get.helm.sh/helm-v${HELM_VERSION}-linux-${ARCHITECTURE}.tar.gz" | \
+    tar --extract --gunzip --strip-components 1 --directory="${install_dir}"/ "linux-${ARCHITECTURE}/helm"
+
+  helm plugin install https://github.com/databus23/helm-diff --version v${HELM_DIFF_VERSION}
+  helm plugin install https://github.com/jkroepke/helm-secrets --version v${HELM_SECRETS_VERSION}
+  helm plugin install https://github.com/aslafy-z/helm-git.git --version v${HELM_GIT_VERSION}
+}
```

Comment:
```suggestion

  su - "${username}" -c "helm plugin install https://github.com/databus23/helm-diff --version v${HELM_DIFF_VERSION}"
  su - "${username}" -c "helm plugin install https://github.com/jkroepke/helm-secrets --version v${HELM_SECRETS_VERSION}"
  su - "${username}" -c "helm plugin install https://github.com/aslafy-z/helm-git.git --version v${HELM_GIT_VERSION}"
}
```

Otherwise the user `jenkins` won't be able to use the plugins (installed in the `root` home dir).

Alternatively: https://github.com/helm/helm/issues/7528#issuecomment-772622281